documentation/SupportingDocuments/API_Documentation/Traffic_Influence_API_Documentation.md

```diff
@@ -308,7 +368,23 @@ N/A
 
 ###  4.8 Release Notes
 
-The Traffic Influence API reduces the complexity of the 3GPP Traffic Influence API exposed by the 3GPP Network Exposure Function (NEF) [1]. While the 3GPP TI API offers fastest routing activation and user mobility among different edge sites, this version of the CAMARA Traffic Influence API covers only the fastest routing activation, also for selected users. User mobility will be introduced in a future version. These release also has no effect on existing PDU sessions.
+The Traffic Influence API reduces the complexity of the 3GPP Traffic Influence API exposed by the 3GPP Network Exposure Function (NEF) [1]. While the 3GPP TI API offers fastest routing activation and user mobility among different edge sites, this version of the CAMARA Traffic Influence API covers only the fastest routing activation, also for selected users. User mobility will be introduced in a future version. 
```

Comment:
I'm not sure that's valuable information for developers, as we are avoiding telco-specific terms where possible. But I agree it is useful information for implementing operators.